src/Controls/samples/Controls.Sample.UITests/Issues/Issue18242.xaml.cs

```diff
@@ -4,7 +4,7 @@
 namespace Maui.Controls.Sample.Issues
 {
 	[XamlCompilation(XamlCompilationOptions.Compile)]
-	[Issue(IssueTracker.Github, 18242, "Button ImageSource not Scaling as expected", PlatformAffected.UWP)]
+	[Issue(IssueTracker.Github, 18242, "Button ImageSource not Scaling as expected", PlatformAffected.All)]
```

Comment:
Could we include here a sample using a different ContentLayout (for example, Top)?
Could we include a case using dynamic size?. We can include a button, and tapping it just increase at runtime the height using a random value.